pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.apache.sling</groupId>
         <artifactId>sling-bundle-parent</artifactId>
-        <version>41</version>
+        <version>49</version>
```

Comment:
why not most recent version 52?

Reply:
Because after a couple of minutes I could not figure out which individual artifacts to replace osgi.core 7.0.0 with. Do you have a suggestion?

Reply:
Usually https://docs.osgi.org/artifacts/#individual-artifacts provides enough information. Just start with framework (https://github.com/apache/sling-parent/blob/b8a8123e95e59762372db37cc1dd2ce15645f7a5/sling-bundle-parent/pom.xml#L242) and then add the following ones from the parent dependency management if needed.

Reply:
That works, thanks. See [b709ec3](https://github.com/apache/sling-org-apache-sling-serviceusermapper/pull/9/commits/b709ec32c710a846affe5f61ab70d726191ba673)